### PR TITLE
Lightly copyedit the text

### PR DESCRIPTION
**Pull request by suchow**
I've been working on a tool that automatically scans prose for errors in usage. I ran it on your book. This pull request fixes some of the issues that it found, including inconsistent spacing at the end of a sentence (rather than choose 1 vs. 2 for the whole book, I made all the pages consistent, one way or the other); the lexical illusion where the word "the" is repeated multiple times; overuse of the word "very", and a few other issues. I hope this is useful to you.

### CODE REVIEW

**Comment by rzach** [comment]
This is great. Thanks! Can you point me to the tool so we can use it again?  (Since it's all LaTeX post-period spacing is irrelevant.)

**Comment by suchow** [comment]
The tool is still in the works, unreleased, but I'll send you a link once it's out. It doesn't yet do autocorrection, only pointing out errors, so I'm afraid I can't switch everything to be single spaced.

File: sets-functions-relations/sets/unions-and-intersections.tex [hunk 1]
```diff
@@ -54,9 +54,9 @@
 \end{ex}
 
 \begin{explain}
-We can obviously also form the union or intersection of more than two
-sets.  An elegant way of dealing with this in general is the
-following: suppose you collect all the sets you want to form the union
+We can also form the union or intersection of more than two
+sets. An elegant way of dealing with this in general is the
+followi))ng: suppose you collect all the sets you want to form the union
```
Comment (rzach) [inline sets-functions-relations/sets/unions-and-intersections.tex]: @suchow this is odd, problem with the script?

**Comment by andrejbauer** [comment]
It would be interesting to see what you catch by running your tool on the [HoTT book](https://github.com/HoTT/HoTT).

**Pull request merged**